frontend/package.json

```
@@ -7,10 +7,11 @@
     "dev": "react-router dev",
     "start": "react-router-serve ./build/server/index.js",
     "typecheck": "react-router typegen && tsc",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "format": "prettier --write ."
   },
   "dependencies": {
-    "@radix-ui/react-slot": "1.2.2",
+    "@radix-ui/react-slot": "^1.2.2",
```

```suggestion
    "@radix-ui/react-slot": "1.2.2",
```
please pin the version